src/jormungandr/jormungandr/src/fragment/pool.rs

```diff
@@ -188,8 +184,10 @@ impl Pool {
 
         let mut fragments = filtered_fragments.into_iter();
         let new_fragments = self.pool.insert_all(fragments.by_ref());
-        let count = new_fragments.len();
-        tracing::debug!("{} of the received fragments were added to the pool", count);
+        tracing::debug!(
+            "{} of the received fragments were added to the pool",
+            new_fragments.len()
+        );
```

Comment:
You could do something like this to make this easier to find and index (eventually):

```suggestion
        tracing::debug!(
            count = %new_fragments.len(),
            "received fragments were added to the pool"
        );
```